src/common-model/entities.ts

```diff
@@ -42,6 +42,7 @@ export enum ClinicalEntitySchemaNames {
 export type EntityAlias =
   | 'donor'
   | 'sampleRegistration'
+  | 'specimen'
   | 'specimens'
```

Comment:
this looks off to me. Shouldn't it just be either 'specimen' or 'specimens'?
looking at current dictionary I think it should be 'specimen' as you've added but then can we safely remove 'specimens'?

Reply:
It is off. The Dictionary defines it as `specimen` but our Donor Model defines it as `specimens`. 
Same issue with `treatment`/`treatments` and several other entities where we have two different sets of aliases.

So I have a lot of code to handle both sets of keys, but it may be redundant in some places and could likely use it's own review.

Reply:
OK w/ the updated validation logic I can remove `specimen` b/c we are checking against schema names

There are other places in the Submitted Data code which should be updated to accept either schema name || donor model aliases and that's what I was trying to address, but that should be dine differently and in another ticket

Reply:
awesome thanks for looking into that